src/modules/verifiable-credentials/storage-client.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  PresentationDefinition,
+  VerifiableCredential,
+  CredentialSubject,
+} from '@ew-did-registry/credentials-interface';
+import { AxiosInstance } from 'axios';
+import { CacheClient } from '../cacheClient';
+
+export default class VCStorageClient {
+  private _http: AxiosInstance;
+
+  constructor(cacheClient: CacheClient) {
+    this._http = cacheClient.http;
+  }
+
+  async getCredentialsByDefinition(
+    definition: PresentationDefinition
+  ): Promise<VerifiableCredential<CredentialSubject>[]> {
+    const { data } = await this._http.get<
+      VerifiableCredential<CredentialSubject>[]
+    >(`/match`, { baseURL: 'vp', params: definition });
```

Comment:
https://github.com/energywebfoundation/ssi-hub/blob/ba3ea1edf48ab1f433c92ad4d13131a7845cad2e/src/modules/presentation-exchange/presentation-exchange.controller.ts#L23
It should be a post method with a body